molecule/util.py

```diff
@@ -202,6 +202,36 @@ def check_ssh_availability(hostip, user, timeout):
         time.sleep(timeout)
         return False
 
+ 
+def generate_temp_ssh_key():
+    # allow python to access the home directory
+    home = expanduser("~")
```

Comment:
Would be nice to remove this duplication. Same logic appears twice in this file, and once in `openstackdriver.py`